app/views/admin/users/_member_index.html.erb

```diff
@@ -54,9 +84,54 @@
         </tr>
       </thead>
       <tbody class="crayons-card">
-        <% @users.each do |user| %>
+        <% @users.includes([:organization_memberships]).each do |user| %>
         <tr>
-          <td><%= link_to user.name, admin_user_path(user) %></td>
+          <td>
+          <div class="flex s:items-start flex-col s:flex-row">
+            <a href="<%= admin_user_path(user.id) %>">
+              <img class="radius-full align-middle" src="<%= user.profile_image_url_for(length: 50) %>" width="40" height="40" alt="<%= user.name %>" />
+            </a>
+            <div class="pl-3 flex-1 fs-s"><%= link_to user.name, admin_user_path(user) %>
+              <p class="color-base-60">@<%= link_to user.username, admin_user_path(user) %>&nbsp;
+                <span class="c-indicator ml-2 crayons-hover-tooltip" data-tooltip="<%= user.current_role %>">
+                  <%= user.current_role %>
+                </span>
+              </p>
+            </div>
+          </div>
+          </td>
+          <td>
+           <% if user.suspended? %>
```

Comment:
maybe with the repetition between this block and the mobile view, we should look to bring it out into its own partial - what do you think (it's not a strong feeling 🙂 )?

Reply:
I agree! However, I couldn't get it right before I signed off yesterday (if we pull it out into a partial, there's a lot of nested looping going on that's not working as hoped 😅 ). I can poke around at it again and worse case, I can always look into it again in a follow-up PR so that we can get this one out in the wild!

Reply:
I added a comment for this on the issue for this work just in case this doesn't work out. I think it'd be fine to do in a follow-up PR if need if!